### DISCUSSION TITLE
Them and syntax hilighting?

### DISCUSSION THREAD

**Discussion by ghost** [sotrh/learn-wgpu#440] [General]
Hi, been following the learn-wgpu tutorials and finding them super helpful, i was just wondering what theme/syntax highlighting plugins plugins you were using for the code examples? I really like the way it highlights different level of module imports, and Ive been trying to set up something similar in vscode.
Thanks again for making this fantastic public resource!

**Comment by sotrh** [comment]
Sorry for the late reply. I'm using PolyMeilex.wgsl for syntax highlighting.
